pkg/ddevapp/config_test.go

```diff
@@ -986,9 +987,7 @@ func TestExtraPackages(t *testing.T) {
 	require.NoError(t, err)
 
 	addedDBPackage := "sudo"
-	// php-gmp is flaky on Debian 12 Bookworm,
-	// we can test it https://github.com/ddev/ddev/issues/5898
-	addedWebPackage := "php" + app.PHPVersion + "-" + "gmp"
```

Comment:
@stasadev I know you put this here for a specific reason when we were having trouble with php-gmp. If you think we should still be testing that situation we should figure out a safer way to test it, probably not in this test.

Reply:
I don't think we will remove the file permissions workaround for the PHP pid file in `/run/php` because it has proven to work well over time.

And the addition of `php-gmp` was done solely to test this workaround. When a test fails due to an unstable connection to deb.sury.org, I don't mind changing the test package here.